### PR TITLE
Feature/quite mode

### PR DESCRIPTION
**Pull request by kagux**

### CODE REVIEW

File: cli.go [hunk 1]
```diff
@@ -22,9 +22,10 @@ const (
 var (
 	app      = kingpin.New("remote_cli", "A remote command line app proxy")
 	isServer = app.Flag("server", "Run in server mode").Short('s').Default("false").Bool()
-	cmd      = app.Flag("cmd", "[Client mode] Command to run").Short('c').String()
 	host     = app.Flag("host", "Host to bind to or to connect to").Short('h').Default("0.0.0.0").String()
 	port     = app.Flag("port", "Port to bind to or to connect to").Short('p').Default("9201").Int()
+	cmd      = app.Flag("cmd", "[Client mode] Command to run").Short('c').String()
+	quite    = app.Flag("quite", "[Client mode] Suppress command output").Short('q').Default("false").Bool()
```
Comment (getapp) [inline cli.go]: typo: quiet

Comment (juanperi) [inline cli.go]: typo: quiet

**Comment by juanperi** [comment]
lgtm

**Comment by juanperi** [comment]
now that I think of it a bit more, the server is still issuing the logs, and the client receives them only to ignore them.
Shouldn't the client send the quiet mode when triggering the remote command, so the server only sends back the final status code?

**Comment by kagux** [comment]
yeah, that would be a good improvement over current implementation

**Pull request merged**